tests/dummy/app/pods/docs/advanced/switching-between-scenarios/template.md

```diff
@@ -0,0 +1,66 @@
+# Switching between scenarios
+
+You can use the scenarios directory to organize your scenario files. Scenarios
+might be useful if you need to conduct [user acceptance testing](https://en.wikipedia.org/wiki/Acceptance_testing#User_acceptance_testing)
+of your application using Mirage data. For example, you might test whether a user 
+can complete a set of tasks specific to unique circumstances defined in a scenario.
+
+Currently, scenarios are not a first-class API: Mirage only uses the default scenario
+in your development environment.
+
+That said, you can still configure your default scenario file to honor options passed
+to the runtime environment. For example, if you created a scenario file called `some-scenario.js`
+and wanted to load that scenario based on an environment variable called `MIRAGE_SCENARIO`, you
+could add the following to `scenarios/default.js`:
+
+```js
+import { camelize } from '@ember/string';
+import ENV from '../../config/environment';
+
+export default function(server) {
+  const { MIRAGE_SCENARIO = '' } = ENV;
+  const scenario = server._config.scenarios[camelize(MIRAGE_SCENARIO)];
+
+  if (scenario) {
+    scenario(server);
+  }
+
+  // plus whatever default scenario code you want
+}
+
+```
+
+You would also need to pull the environment variable into your `environment/config.js`:
+
+```js
+const { MIRAGE_SCENARIO } = process.env;
+
+module.exports = function(environment) {
+  const ENV = {
+    // ...other stuff
+
+    MIRAGE_SCENARIO,
+
+    // ...other stuff
+  };
+
+  return ENV;
+}
+```
+
+Finally, you could switch between scenario files on the command line:
+
+```bash
+MIRAGE_SCENARIO=some-scenario ember s
```

Comment:
Let's move this part first, so as we're explaining the reader knows the starting point. Let's also avoid the private `server._config`.

---

One strategy would be to use ENV vars to switch scenarios. For example, say you wanted to be able to run a scenario file like this:

```
MIRAGE_SCENARIO=administrator ember s
```

To use this `MIRAGE_SCENARIO` ENV variable to alter the behavior of `scenarios/default.js`, add it to your `environment/config.js`:

```js
const { MIRAGE_SCENARIO } = process.env;
module.exports = function(environment) {
  const ENV = {
    // ...other stuff

    MIRAGE_SCENARIO,

    // ...other stuff
  };

  return ENV;
}
```

then access this new config variable in your `scenarios/default.js` file:

```js
// scenarios/default.js
import ENV from '../../config/environment';
import visitor from './visitor';
import subscriber from './subscriber';
import administrator from './administrator';

const scenarios = {
  visitor,
  subscriber,
  administrator
}

const activeScenario = ENV.MIRAGE_SCENARIO || 'visitor';

export default function(server) {
  const scenario = scenarios[activeScenario];

  if (scenario) {
    scenario(server);
  }

  // plus whatever default scenario code you want
}
```